DiscordChatExporter.Cli/Commands/GetChannelsCommand.cs

```diff
@@ -44,6 +50,29 @@ public override async ValueTask ExecuteAsync(IConsole console)
             // Channel category / name
             using (console.WithForegroundColor(ConsoleColor.White))
                 await console.Output.WriteLineAsync($"{channel.Category.Name} / {channel.Name}");
+
+            if (ThreadShow)
+            {
+                var threads = (await Discord.GetGuildChannelThreadsAsync(channel.Id.ToString(), cancellationToken))
+                .OrderBy(c => c.Name)
+                .ToArray();
+
+                foreach (var thread in threads)
+                {
+                    // Thread ID
+                    await console.Output.WriteAsync(
+                        thread.Id.ToString().PadLeft(26, ' ').PadRight(18, ' ')
+                    );
```

Comment:
`PadRight(...)` before was used to ensure the `|` between columns was always in the same place, since channel IDs can actually have different length. Here it's better to just add indentation directly.

```suggestion
                    // Indent
                    await console.Output.WriteAsync('\t');

                    // Thread ID
                    await console.Output.WriteAsync(
                        thread.Id.ToString().PadRight(18, ' ')
                    );
```